script.js

```diff
@@ -0,0 +1,15 @@
+let counter;
+
+if (sessionStorage.getItem('click')) {
+  counter = sessionStorage.getItem('click');
+} else {
+  counter = 0;
+}
+
+document.querySelector('.click').innerHTML = counter;
+window.addEventListener('click', function(){
+  counter++;
+  document.querySelector('.click').innerHTML = counter;
```

Comment:
you duplicated this row.
Create a function for increment `counter` and function for render `counter`